package.json

```diff
@@ -283,6 +283,7 @@
     "fine-uploader": "^5.16.2",
     "foundation-sites": "^5.5.3",
     "history": "3",
+    "history-v4": "npm:history@4.9.0",
```

Comment:
The new version of React Router uses history v4 and we need a way to import that directly in unit tests. Hence, an alias!

Reply:
TIL

`npm install <alias>@npm:<name>:` - https://docs.npmjs.com/cli-commands/install.html